### PR TITLE
Manual testing folder

### PR DESCRIPTION
**Pull request by INCENDE**
- [x] Sample data
- [x] Testing script

### CODE REVIEW

**Comment by burnflare** [comment]
When I checked the sample data yesterday, a lot of the events were very old because the data set was created last week.

Update the dataset so that it's most relevant when looking at it on Wednesday for our demo. Ideally let's only have 1 or 2 max Overdue items, maybe 3-4 due soon items and rest due later items in the left column. Completed tasks on the right column should also look 'nice' when loaded on Wednesday. So let's sample data based on that

**Review by burnflare — APPROVED** [review]

**Comment by fanwgwg** [comment]
@INCENDE can check the test script?

**Comment by INCENDE** [comment]
Script looks good. Can add load and sync

**Comment by INCENDE** [comment]
@burnflare is there a reason the sync tests are failing in this branch?

**Comment by burnflare** [comment]
```
    {
      "code" : 403,
      "errors" : [ {
        "domain" : "usageLimits",
        "message" : "Calendar usage limits exceeded.",
        "reason" : "quotaExceeded"
      } ],
      "message" : "Calendar usage limits exceeded."
    }
```

Damn it, thought this might happen

**Comment by burnflare** [comment]
Google Calendar is rate limiting my test user after it tried to communicate with it too many times.

**Comment by burnflare** [comment]
@INCENDE You can merge this now

File: src/test/data/ManualTesting/testScript.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+*Time Format*
+
+| Time Format     | Example(s)                              |
+|-----------------|-----------------------------------------|
+| Hour            | 10, 22                                  |
+| Hour:Minute     | 10:30                                   |
+| Hour.Minute     | 10.30                                   |
+| Relative time   | this morning, this afternoon, tonight   |
+
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md:50]: update to match user guide again tmr

File: src/test/data/ManualTesting/testScript.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+### 2. Add a task with deadline
+Type `add submit essay by 10pm`.
+#### Result 
+A new task named "submit essay" is created in the **Do It Soon** column, with its deadline under the name the this task. Also, it is highlighted in purple borders.
+
+### 3. Add a task with event time
+Type `add submit essay by 10pm`
+#### Result 
+A new task named "submit essay" is created in the **Do It Soon** column, with its deadline under the name the this task. Also, it is highlighted in purple borders.
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md]: add go for church camp from 20 nov 2pm to 25 nov 5pm
event time not deadline

File: src/test/data/ManualTesting/testScript.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md:78]: to add escape character task

File: src/test/data/ManualTesting/testScript.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+####Result
+Task with index 22 now has no deadline, and it is shifted to **Do It Anytime** column with a new index. Also, it is highlighted in purple borders.
+
+###2. Schedule a task with a deadline
+Type `schedule 23 by 11 nov 10pm`.
+####Result
+Task with index 23 now has a deadline, and it is shifted to **Do It Soon** column with a new index. Also, it is highlighted in purple borders.
+
+###3. Schedule a task with event time
+Type `schedule 14 from 13 nov 2pm to 4pm`.
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md]: schedule 14 from 13 nov 2pm to 13 nov 4pm
(if they check after 13 nov, they might not get the intended result)

File: src/test/data/ManualTesting/testScript.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+####Result
+Task with index 14 now has a event time, and it is assigned with a new index. Also, it is highlighted in purple borders.
+
+## Mark
+To mark a task as completed, you have to start your command with the keyword `mark`.
+
+>Here is the *format*:
+
+>* `mark <id>...` - mark all the tasks identified by `<id>`(s) as completed. Each `<id>` must be a positive number and in the most recent to-do list displayed.
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md]: I will update the main user guide. id are separated by commas or space and can also be inputed as a range

File: src/test/data/ManualTesting/testScript.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+2. The previous marked task is marked again, and the affected task is highlighted in purple borders.
+
+## Find
+>Here is the *format*:
+
+>* `find <keyword>...` - filter out all tasks containing any of the keyword(s) given
+
+Type `find cs2103`
+####result
+Tasks containing `cs2103` in their names will be listed out.
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md:180]: full word, case insensitive

File: src/test/data/ManualTesting/testScript.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+
+>* `alias <original command> <your command>` - `<your command>` must be a single alphanumeric word, and it cannot be a original-command or already aliased to another command. `<original command>` must be a command word that is specified in the Command Summary section
+
+1. Type `alias schedule s`
+2. Type `s 1 by 10pm`
+
+####Result
+1. After the first step, `s` is created as an alias command for `schedule`, and you can now use both `s` and `schedule` to reschedule a task.
+2. After the second step, task with index 1 is rescheduled to be by today 10pm.
+
```
Comment (rachx) [inline src/test/data/ManualTesting/testScript.md:206]: just wondering. are we supposed to show that we handle edge cases
e.g. alias mark add

Comment (fanwgwg) [reply src/test/data/ManualTesting/testScript.md:206]: We already say that the `<your command>` cannot be one of the `<original command>`

**Comment by rachx** [comment]
@fanwgwg please update with the changes
